webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const Path = require('path');
+const webpack = require('webpack');
+// Import the core config
+const webpackConfig = require('@silverstripe/webpack-config');
+const {
+  resolveJS,
+  externalJS,
+  moduleJS,
+  pluginJS,
+  moduleCSS,
+  pluginCSS,
+} = webpackConfig;
+
+const ENV = process.env.NODE_ENV;
+const PATHS = {
+  MODULES: 'node_modules',
+  FILES_PATH: '../',
+  ROOT: Path.resolve(),
+  SRC: Path.resolve('client/src'),
+  DIST: Path.resolve('client/dist'),
+  THIRDPARTY: Path.resolve('thirdparty'),
+};
+
+const config = [
+  {
+    name: 'js',
+    entry: {
+      userforms: `${PATHS.SRC}/bundles/bundle.js`,
+    },
+    output: {
+      path: PATHS.DIST,
+      filename: 'js/[name].js',
+    },
+    devtool: (ENV !== 'production') ? 'source-map' : '',
+    resolve: resolveJS(ENV, PATHS),
+    externals: externalJS(ENV, PATHS),
```

Comment:
hmm, I wonder if you'd need `externalJS`... it's useful for the `admin` interface, but definitely would be more harmful than helpful for the frontend.

I'd suggest creating a separate config group for the frontend with `externalJS` removed/simplified.
We can have a chat about how to do this without a `copy+paste` of the entire `js` block, if you need help :D